docs/source/ml.rst

```diff
@@ -414,3 +423,9 @@ Object reference
 .. autoclass:: civis.ml.ModelFuture
 	       :members:
 	       :inherited-members:
+
+.. autofunction:: civis.ml.put_models_shares_users
+.. autofunction:: civis.ml.put_models_shares_groups
+.. autofunction:: civis.ml.delete_models_shares_users
+.. autofunction:: civis.ml.delete_models_shares_groups
+.. autofunction:: civis.ml.list_models
```

Comment:
Thank you for adding these in! Question -- would it work to put something like `.. automodule:: civis.ml._helper`? If so, that would mean we wouldn't forget to document new functions in that module. The displayed path might be wrong that way, though. Maybe `.. automodule:: civis.ml`? I can't remember if we've tried that before.

Reply:
Yes, I did actually try versions of `.. automodule:: civis.ml._helper`, but no luck. I am 99% sure we had the exact same conversation in the past when we added functions under `civis.io` from time to time but kept forgetting to add the new ones to Sphinx docs. I'm okay with just trying to be more mindful of documentation. :-)

Reply:
Okay, thanks. I thought we must have tried automodule in the past, but I thought I'd double-check.